_includes/markdown/Tools.md

```diff
@@ -26,6 +26,21 @@ At 10up, we use [Vagrant](https://www.vagrantup.com/) to build and interact with
 
 [SVN](https://subversion.apache.org/) - We use SVN, but only in the context of WordPress.com VIP. Again, we encourage people to use the command line as we do not support GUI's internally.
 
+<h3 id="deployments">Deployments {% include Util/top %}</h3>
+The best deployment is quick and requires no site down time. At 10up we’ve spent a lot of our efforts to refine this process and make sure our deployments are secure and speedy. Our 3 primary deployment processes are outlined below
+
+* Atomic
+* Webhooks
+* SFTP uploads
+
+All of these deployment tools are utilizing a remote VCS repository - Beanstalk, Github, Gitlab, etc... 10up's recommended deployment cycle is to trigger deployments on updates to a remote repository. This trigger will start the deployment commands. It's helpful to have an environment already setup to run your build commands. If this is not the case, you will need to install your development dependencies to run your commands at this time, which will slow down your deployment. After you have the development dependencies installed, run your build commands. Once the correct files have been compiled, remove any dependencies. Build dependencies are no longer needed and dependencies are now represented in your compiled files.
```

Comment:
The first part here is a bit hard to follow. I believe we're talking about things that are the same for all of the techniques. We could probably make that it's own paragraph to make that clear. Probably a statement that says more directly deployment methods, no matter what's used, should consider do these things...

The part about dev dependencies should be it's own paragraph as well and stress that devDependencies should not go to production. In fact `node_modules` should not go to production, imho. If we use a front end dependency through NPM and it should not be bundled, the build tool should pull it out of the `node_modules` folder and it should be included from the copied location.